### PR TITLE
build(gamify): small code refactoring + tweaks

### PR DESCRIPTION
**Pull request by mschilling**

### CODE REVIEW

**Review by sanderelderman — CHANGES_REQUESTED** [review]
Rename `libs/gamify/core/src/lib/gamify-api-core.module.ts` to `libs/gamify/core/src/lib/gamify-core.module.ts`

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -13,3 +13,7 @@
 
 # Libs
 libs/ @mschilling @cassshh
+
+# Scope: gamify
+apps/gamify @mschilling @sanderelderman
```
Comment (mschilling) [inline .github/CODEOWNERS:18]: CODEOWNERS file is a fancy thing ;-)

File: libs/gamify/core/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export * from './lib/gamify-core.module';
```
Comment (sanderelderman) [inline libs/gamify/core/src/index.ts:1]: Reference to deleted module file should also be deleted

Comment (mschilling) [reply libs/gamify/core/src/index.ts:1]: How do you mean? I've renamed the import (export)

Comment (sanderelderman) [reply libs/gamify/core/src/index.ts:1]: Well, you kind of deleted the entire file instead of renaming it, don't know if that was on purpose?

**Review by sanderelderman — APPROVED** [review]

File: libs/gamify/core/src/index.ts (hunk 2 above)
Comment (mschilling) [reply libs/gamify/core/src/index.ts:1]: Good catch! I've renamed the file, but the new name wasn't staged properly. No idea why the CI doesn't complain about it 🤷🏽

**Pull request merged**